low-level/actuators/impl/centos_7/systemd_service.py

```diff
@@ -126,42 +123,78 @@ def perform_request(self, jsonMsg):
                 """DisableUnitFiles() function takes second argument as boolean.
                    True will disable a service for runtime only(removes symlink from /run/.. directory)
                    False will disable a service persistently(removes symlink from /etc/.. directory)"""
-                result = self._manager.DisableUnitFiles(service_list, False)
-                self._log_debug("perform_request, result: %s" % result)
-
+                dbus_result = self._manager.DisableUnitFiles(service_list, False)
+                res = parse_enable_disable_dbus_result(dbus_result)
+                result.update(res)
+                logger.debug("perform_request, : %s" % result)
             else:
-                self._log_debug("perform_request, Unknown service request")
-                return (self._service_name, "Unknown service request", None)
+                logger.debug("perform_request, Unknown service request")

```

Comment:
log the unknown request string, with the service name. again keep this in logger.error

Reply:
Done.